### PR TITLE
rename call_type to session_type

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: temba/flows/views.py [hunk 1]
```diff
@@ -1246,7 +1246,7 @@ def get(self, request, *args, **kwargs):
             call = IVRCall.objects.filter(contact__is_test=True, flow=flow).first()
             if call:
                 call = dict(pk=call.pk,
-                            call_type=call.call_type,
+                            session_type=call.session_type,
```
Comment (norkans7) [inline temba/flows/views.py]: I did not see anywhere were we use call_type in the frontend code so I think it is safe to change the key to session_type too

Comment (ericnewcomer) [inline temba/flows/views.py]: There's a few more places, I'll add to this PR

Comment (ericnewcomer) [inline temba/flows/views.py]: I'm inclined that we actually remove it if it's not being used.

**Comment by norkans7** [comment]
@ericnewcomer this is updated

**Pull request merged**